packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -340,11 +346,7 @@ function setInitialDOMProperties(
       } else if (typeof nextProp === 'number') {
         setTextContent(domElement, '' + nextProp);
       }
-    } else if (
-      (enableFlareAPI && propKey === LISTENERS) ||
-      propKey === SUPPRESS_CONTENT_EDITABLE_WARNING ||
-      propKey === SUPPRESS_HYDRATION_WARNING
-    ) {
+    } else if (enableFlareAPIIsSet(propKey)) {
```

Comment:
These aren't all Flare APIs, so the naming is off.

Reply:
would you be kind to suggest a name for the respective variable?

Reply:
@trueadm ?

Reply:
`isInternalPropDefinition`?